docs/source/pages/howToWorkWithPlatform/mainApplication/envelope/approval process/approved.rst

```diff
@@ -0,0 +1,68 @@
+=========================
+Envelope approval process
+=========================
+
+.. toctree::
+
+---------
+
+How to send envelope for approval
+==========================================
+
+
+
+
+
+1.User can send envelope in the "Waiting" status or a draft envelope for approval.Open envelope  in the "Waiting" status, in the upper right corner of the envelope,  click [Approval] button.
+
+.. image:: pic_approval/Screenshot_1.png
+   :width: 600
+   :align: center
+
+
+2.In the field "Аpprover" and "Mailbox" (you can search by the name of the mailbox or uuid code) input details of the approver.Click [Add] button. You can create  unlimited number of approvers.You can delete approver by clicking [Cancel]
+
+.. image:: pic_approval/Screenshot_56.png
+   :width: 600
+   :align: center
+
+
+3.After completing  approval form, click [Confirm] button.
+
+.. image:: pic_approval/Screenshot_45.png
+   :width: 600
+   :align: center
+
+
+4.When you sent the envelope for approval, its status is "On Approval".
+
+..  image:: pic_approval/Screenshot_3.png
+    :width: 600
+    :align: center
+
+
+5.You can view approval process, for this, you need go to envelope with the status "On Approval", select "Approval flow".
+
+.. image:: pic_approval/Screenshot_4.png
+   :width: 600
+   :align: center
+
+
+6.You can view comments and  envelope status.
+
+.. image:: pic_approval/Screenshot_5.png
+   :width: 600
+   :align: center
+
+
+7.When envelope  in the  "On Approval " status,user can reject the envelope.
+
+.. image:: pic_approval/Screenshot_6.png
+   :width: 600
+   :align: center
+
+8.After approval envelope returns to status "Waiting", and user can re-send the envelope for approval.
```

Comment:
8. After approval envelope returns to status "Waiting", and user can re-send the envelope for approval or send it according to the general flow.